cloud/aws/modules/ecs_fargate_service/main.tf

```diff
@@ -110,6 +110,7 @@ moved {
 }
 
 resource "aws_security_group_rule" "ingress_through_http" {
+  count             = var.enable_port_80_listener ? 1 : 0
```

Comment:
(question)  I think we should call it `enable_http_listener`  since we could potentially listen on a different port.

Reply:
I like that better - updated!